app/assets/stylesheets/hitobito/customizable/_fonts.scss

```diff
@@ -0,0 +1,6 @@
+//  Copyright (c) 2021, Pfadibewegung Schweiz. This file is part of
+//  hitobito_pbs and licensed under the Affero General Public License version 3
+//  or later. See the COPYING file at the top-level directory or at
+//  https://github.com/hitobito/hitobito_pbs.
+
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700');
```

Comment:
Sollten wir die Fonts nicht aus Datenschutzgründen lieber direkt in hitobito hosten? In Zukunft führen wir vielleicht mal eine [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) ein, die dann strenger sein kann wenn wir es selber hosten.

Reply:
Wir haben da einfach mal den Core nachgeahmt. Punkto CSP kann ich nicht gross mitreden...

https://github.com/hitobito/hitobito/blob/master/app/javascript/stylesheets/hitobito/customizable/_fonts.scss

Reply:
Wenn das möglich ist die Fonts selbst zu hosten wären wir aber nicht dagegen denke ich.

Reply:
Fonts direkt in hitobito hosten dann aber lieber als neues Issue. Nicht dass wir den PR hier noch abhängig davon machen.